packages/platform-ios/src/commands/runIOS/index.js

```diff
@@ -214,12 +227,11 @@ async function runOnDevice(selectedDevice, scheme, xcodeProject, args: FlagsT) {
   );
 
   if (iosDeployOutput.error) {
-    logger.error(
-      '** INSTALLATION FAILED **\nMake sure you have ios-deploy installed globally.\n(e.g "npm install -g ios-deploy")',
-    );
-  } else {
-    logger.info('** INSTALLATION SUCCEEDED **');
+    throw new Error(`Failed to install the app on the device. We've encountered an error in "ios-deploy" command.
+      Here are more details: ${isIOSDeployInstalled.error.message}`);
```

Comment:
good call with extracting a message. Few improvements:
1. use CLIError (because now it makes sense)
2. change `isIOSDeployInstalled.error.message` to `iosDeployOutput.error.message`
3. change `error in "ios-deploy" command. Here are more details: {message}` to just `error in "ios-deploy"  command: {message}` so it's more concise